### DISCUSSION TITLE
New NextJS 14 application, Vercel deployment fails on `npm i` without using private packages: Incorrect or missing password

### DISCUSSION THREAD

**Discussion by albertnev** [vercel/community#7218] [Help] [answered]
### Summary

Hello,

I'm a bit lost here, since this is the first app with the new version 14 of NextJS I've made and I don't know if there are some new requirements or parameters needed that have changed with the older versions.

Thing is, I'm using NextJS14, Tailwind and Typescript, and I haven't used any private packages. When I run the `npm run build` command in local, it works. However, as soon as I make a commit and try to deploy it on Vercel, it shows this error:

```
[18:00:37.893] Running build in Washington, D.C., USA (East) – iad1
[#################]
[18:00:38.320] Skipping build cache, deployment was triggered without cache.
[18:00:39.165] Cloning completed: 866.275ms
[18:00:39.838] Running "vercel build"
[18:00:40.695] Vercel CLI 34.2.7
[18:00:41.613] Running "install" command: `npm install`...
[18:00:52.129] npm ERR! code E401
[18:00:52.130] npm ERR! Incorrect or missing password.
[18:00:52.130] npm ERR! If you were trying to login, change your password, create an
[18:00:52.130] npm ERR! authentication token or enable two-factor authentication then
[18:00:52.130] npm ERR! that means you likely typed your password in incorrectly.
[18:00:52.130] npm ERR! Please try again, or recover your password at:
[18:00:52.130] npm ERR!     https://www.npmjs.com/forgot
[18:00:52.130] npm ERR! 
[18:00:52.130] npm ERR! If you were doing some other operation then your saved credentials are
[18:00:52.130] npm ERR! probably out of date. To correct this please try logging in again with:
[18:00:52.130] npm ERR!     npm login
```
(I've deleted sensitive information)

I've tried to delete `package-lock.json` and re-install (and commit changes) without luck. I've also tried overriding the install and build commands with the ones from my `package.json`, also without luck. I had my GitHub repo in private, so I changed it to public, but that didn't help either.

Any help or any guide will be much appreciated, as I don't know why it's failing because the error doesn't give me any hints (a part from the "wrong credentials" to install public packages). 

**Thank you so much!**

### Steps to Reproduce

These are my `package.json` `dependencies` and `devDependencies`. I just try to deploy my GitHub:
```
"dependencies": {
    "@types/jest": "^29.5.12",
    "clsx": "^2.1.1",
    "next": "14.2.3",
    "react": "^18",
    "react-dom": "^18",
    "react-icons": "^5.2.1",
    "react-intersection-observer": "^9.10.3"
  },
  "devDependencies": {
    "@next/eslint-plugin-next": "^14.2.3",
    "@playwright/test": "^1.44.1",
    "@testing-library/jest-dom": "^6.4.5",
    "@testing-library/react": "^16.0.0",
    "@testing-library/user-event": "^14.5.2",
    "@types/node": "^20.14.2",
    "@types/react": "^18",
    "@types/react-dom": "^18",
    "@vercel/style-guide": "^6.0.0",
    "@vitejs/plugin-react": "^4.3.1",
    "eslint": "^8.2.0",
    "eslint-config-next": "14.2.3",
    "eslint-config-prettier": "^9.1.0",
    "eslint-plugin-jsx-a11y": "^6.8.0",
    "eslint-plugin-sort-destructure-keys": "^2.0.0",
    "eslint-plugin-typescript-sort-keys": "^3.2.0",
    "jsdom": "^24.1.0",
    "postcss": "^8",
    "prettier": "^3.3.1",
    "tailwindcss": "^3.4.1",
    "tailwindcss-animate": "^1.0.7",
    "typescript": "^5",
    "vitest": "^1.6.0"
  }
```

**Comment by versecafe** [comment]
@albertnev swap `"@next/eslint-plugin-next": "^14.2.3"` to `eslint-plugin-next": "^14.2.3"`

**Reply by albertnev** [reply]
Install process doesn't work, not even locally. It doesn't find the package, and in their official documentation they make you install it with the "@next/" part and [there is no other valid package without the "@next/" part](https://www.npmjs.com/package/eslint-plugin-next).

**Reply by versecafe** [reply]
https://www.npmjs.com/package/eslint-config-next

<img width="1287" alt="image" src="https://github.com/vercel/community/assets/147033096/b4a7dfda-2bba-4a77-b029-cb5d7be105a5">

No @next prefix should exist for the package

**Reply by albertnev** [reply]
I'm sorry, but I'm not sure I follow. The screenshot you're showing is for `eslint-config-prettier`. The package you mentioned before, which is the one that has @next in its name, is `@next/eslint-plugin-next`, which is a different package. 

In the `.eslintrc.js` file I'm referencing the right packages (otherwise eslint wouldn't work in local, either), but ESLint should not affect the `npm install` command, as it needs to first install dependencies to run ESLint later.

**Comment by swarnava** [comment, marked as answer]
Closing in favour of https://github.com/orgs/vercel/discussions/7237

[a comment by an automated account is not shown]
